record_indexer/index_templates/record_index_config.json

```diff
@@ -57,23 +57,10 @@
                 "repository_data": {"type": "keyword"},
                 "repository_url": {"type": "keyword"},
                 "rights_uri": {"type": "keyword"},
-                "reference_image_md5": {"type": "keyword"},
-                "reference_image_dimensions": {"type": "keyword"},
-                "manifest": {"type": "keyword"},
-                "object_template": {"type": "keyword"},
-                "url_item": {"type": "keyword"},
```

Comment:
Let's keep url_item - the Calisphere UI uses it: https://github.com/search?q=repo%3Aucldc%2Fpublic_interface%20url_item&type=code

`solr_updater` populates it from `is_shown_at`. Pretty sure calisphere_solr_mapper collections will still use the `solr_updater`, right? So it's good to remove from the mapper, but we still want it in the index.

Reply:
Yup, thanks for catching this!!